src/utils/getContributedRepos.ts

```diff
@@ -0,0 +1,24 @@
+interface PRResponse {
+    data: [{
+        full_name: string;
+    }]
+    meta: {
+        itemCount: number;
+    };
+}
+
+export const countUniqueRepos = (response: PRResponse | null) => {
+    if (!response?.data) {
+        return 0;
+    }
+
+    const { data } = response;
+    const uniqueRepos = (new Set<string>);
+
+    // eslint-disable-next-line no-loops/no-loops
+    for (const obj of data) {
+        uniqueRepos.add(obj.full_name);
+    }
```

Comment:
```suggestion
const uniqueRepos = new Set(data.map(obj => obj.full_name));
```

_Generated using [OpenSauced](https://opensauced.ai/)._

Looooks correct, good AI. Let me know if it works.

Reply:
i was gonna use this one
const uniqueFullNames = data.reduce((uniqueNames: Set<string>, obj) => {
                            uniqueNames.add(obj.full_name);
                            return uniqueNames;
                        }, new Set<string>());

but both work well

Reply:
yeah, either is fine, the first one is more readable though, I prefer not using .reduce wherever possible.